build-logic/src/main/groovy/org/apache/groovy/gradle/JarJarTask.groovy

```diff
@@ -112,7 +114,7 @@ class JarJarTask extends DefaultTask {
         def manifestFile = new File(temporaryDir, 'MANIFEST.MF')
         // First step is to create a repackaged jar
         outputFile.parentFile.mkdirs()
-        def tstamp = Date.parse('yyyy-MM-dd HH:mm', '1980-02-01 00:00').time.toString()
+        def tstamp = (new SimpleDateFormat('yyyy-MM-dd HH:mm')).parse('1980-02-01 00:00').time.toString()
```

Comment:
The original is more idiomatic Groovy

Reply:
Cause `Date.parse` is deprecated.

Reply:
My bad, it might be a bug of IDEA, actually no deprecation on `parse`, will revert.